eclair-node/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -40,6 +42,10 @@ object Boot extends App with Logging {
     implicit val system: ActorSystem = ActorSystem("eclair-node")
     implicit val ec: ExecutionContext = system.dispatcher
     val setup = new Setup(datadir)
+
+    // TODO: disable Kamon for now
+    //Kamon.init(setup.appConfig)
```

Comment:
Might be worth making this configurable?

Reply:
Yes, I was just being lazy